packages/@magic-ext/connect/README.md

```diff
@@ -20,10 +20,14 @@ Integrating your app with Magic will require our client-side NPM package and the
 
 ```bash
 # Via NPM:
-npm install --save magic-sdk @magic-ext/connect
+npm install --save magic-sdk @magic-ext/connect # If you're targeting web browsers
```

Comment:
What do you think we reformat this to

Browser
```
yarn
```

bare RN
```
yarn
```

bare Expo
```
yarn
```

I think this format will provide more clarity on platform selections